lang/cs/Org.Apache.REEF.Wake/Org.Apache.REEF.Wake.DotNet.csproj

```diff
@@ -0,0 +1,37 @@
+<Project Sdk="Microsoft.NET.Sdk">
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+  <PropertyGroup>
+    <AssemblyName>Org.Apache.REEF.Wake</AssemblyName>
+    <Description>Wake is an event driven framework</Description>
+    <PackageTags>wake event driven</PackageTags>
```

Comment:
The tags show up in nuget packages and currently event and driven show up as two different tags which is confusing. Other packages us event-driven. Maybe we should switch to this tag also?